### PR TITLE
update guac-visualizer readme for v0.2.0

### PR DESCRIPTION
**Pull request by shafeeshafee**
- Updates the doc to reflect usage of guac v0.2.0 with visualizer v0.2.0
- Added updated image

### CODE REVIEW

File: guac-visualizer.md [hunk 1]
```diff
@@ -27,83 +27,56 @@ well as a means to explore the supply chain and prototype policies.
...
-Change directories into the visualizer's repo:
-
-```bash
-cd path/to/guac-visualizer
-```
+### 1. Download GUAC version `v0.2.0` from [here](https://github.com/guacsec/guac/releases/tag/v0.2.0).
 
-All commands run throughout this guide should be in this working directory.
+- Follow these steps outlined here to get the GUAC server up and running:
+  https://docs.guac.sh/setup
```
Comment (jeffmendoza) [inline guac-visualizer.md:42]: Should be linked like this `[link text here]({{ site.baseurl }}{% link setup.md %})`

File: guac-visualizer.md [hunk 2]
```diff
@@ -27,83 +27,56 @@ well as a means to explore the supply chain and prototype policies.
...
-   [this link](https://github.com/guacsec/guac-visualizer/releases/tag/v0.1.1)
-   and extract it in the same path as your GUAC repo.
+To use the GUAC visualizer, you need to have the main GUAC server running.
 
-Change directories into the visualizer's repo:
-
-```bash
-cd path/to/guac-visualizer
-```
+### 1. Download GUAC version `v0.2.0` from [here](https://github.com/guacsec/guac/releases/tag/v0.2.0).
```
Comment (jeffmendoza) [inline guac-visualizer.md:39]: Probably don't link to the release here and just have the link to setup.md. Just saying "Download" it's not clear what to download.

**Comment by jeffmendoza** [comment]
Some things look a little weird, see below (I had some fun with paint)
image

The blue steps 1, 2, 3 are too big, did you mean to put them to title-size (single #) ?

Having the blue steps and red steps as both numbers doesn't make sense, either remove the numbers from the red steps, to make them one big step, or change to A and B.

The url underlined in orange doesn't seem to be a link.

We say to run guac twice, green underline, and red step 1.

I'm not sure the part where we say "same path" makes sense. Will people extract it into the guac repo? Maybe we don't even need to say that.

File: guac-visualizer.md [hunk 3]
```diff
@@ -26,90 +26,50 @@ well as a means to explore the supply chain and prototype policies.
...
-- A fresh copy of the [GUAC service infrastructure through Docker Compose]({{
-  site.baseurl }}{%link setup.md %}).
+- [git](https://git-scm.com/downloads)
+- [yarn](https://classic.yarnpkg.com/lang/en/docs/install/#mac-stable)
+- [node](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm)
+- [GUAC services up and running]({{
+  site.baseurl }}{%link setup.md %})
 
-## Step 1: Getting started
+## Step 1. Getting started
```
Comment (pxp928) [inline guac-visualizer.md:35]: @jeffmendoza should we move this into the setup section along with the docker-compose install?

**Review by jeffmendoza — APPROVED** [review]

**Pull request merged**